core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/deploy/maven/AbstractMavenDeployer.java

```diff
@@ -59,6 +59,7 @@ public abstract class AbstractMavenDeployer<S extends AbstractMavenDeployer<S, A
     protected Boolean javadocJar;
     protected Boolean verifyPom;
     protected Boolean applyMavenCentralRules;
+    protected Boolean mayDeploySnapshots;
```

Comment:
Please rename to `snapshotSupported` otherwise json-schema may not pick up the correct property name 😓

Reply:
Set the initial value to `false` to keep BC.

Reply:
Will do. I was using the name in question since it's extremely hard to keep track of what "setIsSnapshotSupoprtedSet" etc in English.